sensor_msgs/Image previous_image
sensor_msgs/Image current_image

---
sensor_msgs/Image output_image
sensor_msgs/Image output_viz
bool success